packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.hooks.tsx

```diff
@@ -173,6 +174,7 @@ export const useYppGoogleAuth = ({
         if (isAxiosError<ChannelVerificationErrorResponse>(error)) {
           const errorResponseData = error.response?.data
           const errorMessages = error.response?.data.message
+          const channelRegisteredData = errorResponseData as ChannelAlreadyRegisteredError
```

Comment:
I don't see the point of reassigning a variable to just do the assertion. You could add an assertion on line `222`

```tsx
 variables: { where: { id: (errorResponseData as ChannelAlreadyRegisteredError).result.toString() } },
```

Also, do we actually need the type assertion?